src/main/java/uk/gov/companieshouse/insolvency/data/config/ApplicationConfig.java

```diff
@@ -23,9 +37,32 @@ EnvironmentReader environmentReader() {
         return new EnvironmentReaderImpl();
     }
 
+    @Bean
+    @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+    public InternalApiClient internalApiClient() {
+        return ApiSdkManager.getPrivateSDK();
+    }
+
+    /**
+     * mongoCustomConversions.
+     * @return MongoCustomConversions.
+     */
     @Bean
     public MongoCustomConversions mongoCustomConversions() {
-        return new MongoCustomConversions(List.of(insolvencyConverter));
+        ObjectMapper objectMapper = mongoDbObjectMapper();
+        return new MongoCustomConversions(List.of(new CompanyInsolvencyConverter(objectMapper),
+                new CompanyInsolvencyReadConverter(objectMapper)));
     }
 
+    private ObjectMapper mongoDbObjectMapper() {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        objectMapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+        SimpleModule module = new SimpleModule();
+        module.addSerializer(LocalDate.class, new LocalDateSerializer());
+        module.addDeserializer(LocalDate.class, new LocalDateDeSerializer());
```

Comment:
We probably need serialisers/de-serialisers for OffsetDateTime as well to store as IsoDate into DB